src/main/java/frc/robot/RobotMap.java

```diff
@@ -15,6 +15,17 @@ public final class RobotMap {
     REAR_LEFT_TURNING_CAN_ID = 12,
     FRONT_RIGHT_TURNING_CAN_ID = 14,
     REAR_RIGHT_TURNING_CAN_ID = 16;
+
+  /** Intake deployment pneumatic channels */
+  public static final int INTAKE_DEPLOY_LEFT_FORWARD_CHANNEL = 0, 
+  INTAKE_DEPLOY_LEFT_REVERSE_CHANNEL = 1, 
+  INTAKE_DEPLOY_RIGHT_FORWARD_CHANNEL = 2, 
+  INTAKE_DEPLOY_RIGHT_REVERSE_CHANNEL = 3;
+
+  
+  /** Intake Spark MAX CAN IDs */
+  public static final int INTAKE_LEFT_MOTOR_CAN_ID = 22,
+  INTAKE_RIGHT_MOTOR_CAN_ID = 23;
```

Comment:
I've been indenting the subsequent variables in each chunk - see lines 14-17 for an example. I do not care *whatsoever* either way, but let's be consistent. I like consistency!

Reply:
i see!